www/ApptentiveIos.js

```diff
@@ -75,6 +75,10 @@ var Apptentive = {
         cordova.exec(unreadMessagesCallback, errorCallback, "ApptentiveBridge", "execute", ["registerForMessageNotifications"]);
     },
 
+    addSurveyFinishedListener: function (surveyFinishedCallback, errorCallback) {
+        errorCallback(); // Does nothing on iOS.
+    },
```

Comment:
@frankus Do you want to add the functionality for this and sendAttachmentText for iOS for the Cordova app?

Reply:
I could. I also have a story in the sprint with debug mode.

I'll branch off of your work here and jump on that today.

Reply:
Added in https://github.com/apptentive/apptentive-cordova/pull/99